jaeger-dropwizard/src/main/java/com/uber/jaeger/dropwizard/Configuration.java

```diff
@@ -45,7 +45,7 @@ public Configuration(
   }
 
   @Override
-  public Tracer getTracer() {
+  synchronized public Tracer getTracer() {
```

Comment:
I think it is because this class inherits from `jaeger/Configuration.java` in which I had declared `getTracer()` to be synchronized